crates/iota-protocol-config/src/lib.rs

```diff
@@ -1679,6 +1689,7 @@ impl ProtocolConfig {
         cfg.feature_flags.consensus_round_prober = false;
         cfg.feature_flags
             .consensus_distributed_vote_scoring_strategy = false;
+        cfg.feature_flags.congestion_control_min_free_execution_slot = false;
```

Comment:
This (and the feature_flag bools above) aren't needed, as it's false by default
I also commented this in another PR some time ago but unfortunately don't know what happened with it

Reply:
@muXxer any reason to add it, even if it's default?

Reply:
Yeah ok, I looked at the sui codebase again.

As far as I could remember, they had them sometimes mentioned and set to false, and sometimes not mentioned, but I couldn't find those cases anymore. Maybe that was only the case in our first codebase, and thats where I decided to "then lets always do the same and mention them and set them to false". 

But ok, I guess removing is better. Can someone remove all the ones that are set to false there, or should we do this in another PR?

Reply:
I will remove the line for our new feature flag in this PR, and here is a PR to remove the others https://github.com/iotaledger/iota/pull/6541